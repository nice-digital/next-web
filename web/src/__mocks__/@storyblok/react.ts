export const storyblokInit = (): null => null;

export const apiPlugin = (): null => null;
