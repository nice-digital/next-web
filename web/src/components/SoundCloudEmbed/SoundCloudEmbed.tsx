interface SoundCloudEmbedProps {
	id: string;
}

export const SoundCloudEmbed: React.FC<SoundCloudEmbedProps> = ({
	id,
}: SoundCloudEmbedProps) => (
	<iframe
		title="SoundCloud player for this NICE podcast"
		width="100%"
		height="200"
		scrolling="no"
		frameBorder="no"
		src={`https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/${id}&color=%23ff5500&auto_play=false&hide_related=true&show_comments=false&show_user=true&show_reposts=false&show_teaser=true&visual=true`}
	></iframe>
);
