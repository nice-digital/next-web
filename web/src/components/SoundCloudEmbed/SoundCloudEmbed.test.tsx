import { render } from "@testing-library/react";

import { SoundCloudEmbed } from "./SoundCloudEmbed";

describe("SoundCloud embed component", () => {
	it("should match snapshot", () => {
		const { container } = render(<SoundCloudEmbed id="787280485" />);
		expect(container).toMatchSnapshot();
	});
});
