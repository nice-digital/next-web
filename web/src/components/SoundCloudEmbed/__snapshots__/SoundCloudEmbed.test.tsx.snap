// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SoundCloud embed component should match snapshot 1`] = `
<div>
  <iframe
    frameborder="no"
    height="200"
    scrolling="no"
    src="https://w.soundcloud.com/player/?url=https%3A//api.soundcloud.com/tracks/787280485&color=%23ff5500&auto_play=false&hide_related=true&show_comments=false&show_user=true&show_reposts=false&show_teaser=true&visual=true"
    width="100%"
  />
</div>
`;
